src/main/java/org/reaktivity/command/log/internal/LogQueueDepthCommand.java

```diff
@@ -69,22 +63,24 @@ private void onDiscovered(
         }
     }
 
-    private void bufferCapacity(
+    private void displayQueueDepth(
             Path path)
     {
-        StreamsLayout layout = new StreamsLayout.Builder()
+        try (StreamsLayout layout = new StreamsLayout.Builder()
                 .path(path)
                 .streamsCapacity(streamsCapacity)
                 .throttleCapacity(throttleCapacity)
                 .readonly(true)
                 .build();
-        DirectBuffer buffer = layout.streamsBuffer().buffer();
-        System.out.println(String.format(
-                logformat,
-                directory.relativize(path),
-                buffer.getLong(buffer.capacity()  - TRAILER_LENGTH + HEAD_POSITION_OFFSET),
-                buffer.getLong(buffer.capacity() - TRAILER_LENGTH + TAIL_POSITION_OFFSET),
-                buffer.capacity()));
+        )
+        {
+            DirectBuffer buffer = layout.streamsBuffer().buffer();
+
+            System.out.printf("%s readPointer=%d writePointer=%d \n",
+                    directory.relativize(path),
+                    layout.streamsBuffer().consumerPosition(buffer),
+                    layout.streamsBuffer().producerPosition(buffer));
```

Comment:
Need to present the queue depth, which would be the difference between consumer and producer positions.